configure.ac

```diff
@@ -228,7 +228,9 @@ AM_CONDITIONAL([ENABLE_FUZZING],[test "x$with_fuzzing" != "xnone"])
 AS_IF([test "x$with_fuzzing" != "xnone"],
       [ADD_COMPILER_FLAG([-fsanitize=fuzzer-no-link], [required])
        AS_IF([test "x$enable_tcti_fuzzng" = "xno"],
-             AC_MSG_ERROR([Fuzz tests can not be enabled without the TCTI_FUZZING module]))])
+             AC_MSG_ERROR([Fuzz tests can not be enabled without the TCTI_FUZZING module]))
+       AS_IF([test "x$GEN_FUZZ" != "x1"],
+             AC_MSG_ERROR([Fuzz tests can not be enabled without "GEN_FUZZ=1" variable]))])
```

Comment:
`GEN_FUZZ` env variable is only used in bootstrap to see if it needs to generate the cpp and makefile

Reply:
The thing it that you can run ./bootstrap without setting GEN_FUZZ and then run ./configure --enable_tcti_fuzzng=yes and thing don't work. This check is a best attempt to ensure that if you run configure --enable_tcti_fuzzng=yes the fuzz.cpp files are actually generated.

Reply:
Calling
```
GEN_FUZZ=1 ./bootstrap
./configure --with-fuzzing=libfuzzer --enable-tcti-fuzzing --disable-tcti-device --disable-tcti-mssim
```
should be valid, but fails, as during ./configure the variable is not set.  Rather than checking whether the GEN_FUZZ variable is set to one, ./configure shall validate, that the file $(top_buiddir)/test/fuzz/Tss2_Sys_Clear_Complete.fuzz.cpp exists, e.g. with `AC_CHECK_FILE()`.

I do not plan to provide a PR.